Command/GenerateBundleCommand.php

```diff
@@ -150,7 +150,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
                 'See http://symfony.com/doc/current/cookbook/bundles/best_practices.html#bundle-name for more',
                 'details on bundle naming conventions.',
                 '',
-                'Use <comment>/</comment> instead of <comment>\\ </comment> for the namespace delimiter to avoid any problem.',
+                'Use <comment>/</comment> instead of <comment>\\</comment> for the namespace delimiter to avoid any problems.',
```

Comment:
the space after the backslash should be reverted and the one before `for` removed instead, or `<` will be escaped breaking the style.